tests/schema.py

```diff
@@ -30,7 +30,9 @@ def run(args):
     documents_valid = True
     all_methods = []
 
-    def fetch_schema(client, prefix):
+    def fetch_schema(client, prefix, file_prefix = None):
```

Comment:
Sorry these changes are unrelated, and part of an abandoned change. I noticed we don't fetch the API for any JS apps, so was adding that here. But it turns out they're generally _not_ valid OpenAPI yet. So we can look at implementing that separately.